src/typeDefs/schema.graphql

```diff
@@ -354,19 +358,18 @@ enum Specialty {
 # ======== Categories for specialties =========
 
 enum SpecialtyCategory {
-  PRIMARY_CARE
-  INTERNAL_MEDICINE
-  SURGICAL
-  DIAGNOSTIC
-  MENTAL_HEALTH
-  WOMENS_HEALTH
   CHILDRENS_HEALTH
-  MUSCULOSKELETAL
-  EYE_AND_VISION
+  COSMETIC_AND_PLASTIC_SURGERY
   DENTAL
+  DERMATOLOGY
   ENT
+  EYE_AND_VISION
+  MENTAL_HEALTH
+  PHARMACY
```

Comment:
```suggestion
  PHARMACY
```

I think we should remove this one for now since it's not a hospital or clinic. Let's talk it over with the board first.